package main

import "testing"

func TestPresentation(_ *testing.T) {
	// If there is an error in main this test will fail
	main()
}
